ui/src/components/domain/ManageDomains.js

```diff
@@ -401,6 +406,15 @@ class ManageDomains extends React.Component {
                                   />
                               ) : null}
                           </TDStyled>
+
+                          <TDStyled color={color} align={center}>
+                              {item.domain.account ? item.domain.account : ''}
```

Comment:
what is this account after the delete icon?

Reply:
it has to do with the alignment of the "account" column.

Reply:
we already display account in line 380, not the right way to align columns

Reply:
I see that it was already mentioned in the earlier lines, I fixed the error.